src/test/java/eu/elixir/ega/ebi/it/res/FileControllerIntgTest.java

```diff
@@ -0,0 +1,95 @@
+package eu.elixir.ega.ebi.it.res;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.math.BigInteger;
+import java.security.DigestInputStream;
+import java.security.MessageDigest;
+
+import org.apache.http.HttpStatus;
+import org.json.JSONObject;
+import org.junit.Test;
+
+import com.google.common.io.CountingInputStream;

```

Comment:
Unused import

Reply:
Also, not sure if the guava library is needed anymore now

Reply:
removed the import and also from pom.xml